packages/patternfly-3/patternfly-react-extensions/src/components/TableGrid/TableGridRow.js

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import { Grid } from 'patternfly-react';
+import { Grid, noop } from 'patternfly-react';
 
 /**
  * TableGridRow Component for PatternFly
  */
 
-const TableGridRow = ({ children, className, selected, ...props }) => {
+const TableGridRow = ({ children, className, selected, onToggleSelection, ...props }) => {
   const classes = classNames('table-grid-pf-row', { active: selected }, className);
   return (
     <Grid.Row className={classes} {...props}>
+      <div className="table-grid-pf-checkbox">
+        <input type="checkbox" checked={selected} onChange={onToggleSelection} />
```

Comment:
we may want to also consider rendering `label` elements alongside `input` elements with an `sr-only` class applied to labels. (comparing/contrasting w/ existing table view/checkboxes)